src/rust/inetstack/protocols/udp/tests.rs

```diff
@@ -106,8 +106,8 @@ fn udp_push_pop() -> Result<()> {
     };
     now += Duration::from_micros(1);
 
-    // Receive data from Bob.
-    carrie.receive(bob.pop_frame()).unwrap();
+    // Receive data from Bob and insert into Carrie's incoming queue.
+    carrie.push_frame(bob.pop_frame());
```

Comment:
Why is this comment different from comments at other instances of the same code? Is this a special case?